app/views/item_categories/show.html.erb

```diff
@@ -0,0 +1,87 @@
+<section class="content-header">
```

Comment:
Should there be a button to the edit page somewhere here? Otherwise you have to go back to the index page to click edit

Reply:
Hmm good point! I think that should be a simple add.